src/index.ts

```diff
@@ -164,6 +185,13 @@ function removeNullOrUndef(params: ParamMap) {
     }, {} as ParamMap);
 }
 
+/**
+ * Checks whether :param is Null  or undefined.
+ * @param v {any} :param
+ *
+ * @returns {Boolean} true -> :param is not Null or not undefined
+ * @returns {Boolean} false -> :param is Null or undefined
+ */
```

Comment:
This is an internal function that isn't exported so it doesn't need a jsdoc string.